src/lower.c

```diff
@@ -1921,15 +1921,16 @@ static void lower_logic_expr_coverage(lower_unit_t *lu, tree_t fcall,
 
    flags |= COV_FLAG_EXPR_STD_LOGIC;
 
-   assert(tree_params(fcall) == 2);
-
-   bool is_array[2] = {
-      type_is_array(type_param(tree_type(decl), 0)),
-      type_is_array(type_param(tree_type(decl), 1))
-   };
+   const unsigned nparams = tree_params(fcall);
+   assert(nparams > 0);
```

Comment:
```suggestion
   if (nparams != 2)
      return;
```

Since we don't care about anything except two-argument calls.

Could you also delete the call to `fmt_loc` on line 1916? I think I left that in while I was debugging something a few weeks ago.